com.palantir.typescript/src/com/palantir/typescript/IPreferenceConstants.java

```diff
@@ -41,6 +41,7 @@
     String COMPILER_OUT_FILE = "compiler.outputFileOption";
     String COMPILER_REMOVE_COMMENTS = "compiler.removeComments";
     String COMPILER_SUPPRESS_IMPLICIT_ANY_INDEX_ERRORS = "compiler.suppressImplicitAnyIndexErrors";
+    String COMPILER_SUPPRESS_EXCESS_PROPERTY_ERRORS = "compiler.suppressExcessPropertyErrors";
```

Comment:
very cool - can i ask for one small change across the patch? i typically sort things so could you put the new property above suppressImplicitAnyIndexErrors all over?